crates/cdk-axum/src/ws/unsubscribe.rs

```diff
@@ -0,0 +1,29 @@
+use super::{handler::WsHandle, WsContext, WsError};
+use cdk::pub_sub::SubId;
+
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+pub struct Method {
+    pub sub_id: SubId,
+}
```

Comment:
```suggestion
pub struct Method {
    #[serde(rename = "subId")]
    pub sub_id: SubId,
}
```

sub id should be snake case